main.go

```diff
@@ -83,20 +83,18 @@ func main() {
 	printUserInfo()
 
 	var db *gorm.DB
-	for i := 1; i <= configuration.GetPostgresConnectionMaxRetries(); i++ {
-		log.Printf("Opening DB connection attempt %d of %d\n", i, configuration.GetPostgresConnectionMaxRetries())
+	for {
 		db, err = gorm.Open("postgres", configuration.GetPostgresConfigString())
 		if err != nil {
 			db.Close()
+			log.Printf("ERROR: Unable to open connection to database %v\n", err)
+			log.Printf("Retrying to connect in %v...\n", err, configuration.GetPostgresConnectionRetrySleep())
```

Comment:
Caught by your own checks... 
/tmp/go/src/github.com/almighty/almighty-core/main.go:91: wrong number of args for format in Printf call: 1 needed but 2 args

:D